src/TodoListItem.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const TodoListItem = (props) => {
+  return (
+    <div>
+      <li>{props.title}</li>
+    </div>
+  );
```

Comment:
You did really well here. 

As an improvement, you can destructure the props like this:

```
const TodoListItem = ({title}) => {
return (
    <div>
      <li>{title}</li>
    </div>
  );
  }
  ```
  
  This way you don't have to write out `props.whatever`. Let me know if this is clear